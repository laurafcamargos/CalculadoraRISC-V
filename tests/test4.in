2
*
3
u
5
f
