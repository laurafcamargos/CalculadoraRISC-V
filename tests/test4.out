Digite um número: Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 6
Digite a operação (+, -, *, /, u, f): Operação desfeita. Nenhum resultado salvo.
Digite um número: Digite a operação (+, -, *, /, u, f): 
